ui-react/packages/atlasmap/src/Atlasmap/utils/ui.ts

```diff
@@ -160,8 +160,8 @@ export function fromMappingDefinitionToIMappings(
 export function executeFieldSearch(
   searchFilter: string,
   isSource: boolean,
-): void {
-  initializationService.cfg.mappingService.executeFieldSearch(
+): any[] {
```

Comment:
Please never use `any`, it removes the benefit of using Typescript in first place. If we don't need the return of the `executeFieldSearch` method then we shouldn't return it.

Reply:
We do need the return of executeFieldSearch - It sends back an array of { displayName and a field node}.   I'm looking to see if I can send back just an array of strings.